src/Nethermind/Nethermind.Trie/Pruning/RawScopedTrieStore.cs

```diff
@@ -0,0 +1,62 @@
+// SPDX-FileCopyrightText: 2025 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using Nethermind.Core;
+using Nethermind.Core.Crypto;
+
+namespace Nethermind.Trie.Pruning;
+
+public class RawScopedTrieStore(INodeStorage nodeStorage, Hash256 address) : IScopedTrieStore
```

Comment:
I've seen it used a lot, so maybe:

```suggestion
public class RawScopedTrieStore(INodeStorage nodeStorage, Hash256? address = null) : IScopedTrieStore
```